crates/rpc/src/v05/method/get_transaction_receipt.rs

```diff
@@ -156,9 +160,63 @@ pub mod types {
         #[serde(skip_serializing_if = "Option::is_none")]
         pub revert_reason: Option<String>,
         pub execution_status: ExecutionStatus,
+        #[serde(skip_serializing_if = "Option::is_none")]
+        pub execution_resources: Option<ExecutionResourcesProperties>,
         pub finality_status: FinalityStatus,
     }
 
+    /// Similar to `ExecutionResources`, with irrelevant properties stripped.
+    #[serde_as]
+    #[derive(Clone, Debug, Serialize, PartialEq, Eq)]
+    #[cfg_attr(any(test, feature = "rpc-full-serde"), derive(serde::Deserialize))]
+    pub struct ExecutionResourcesProperties {
+        // TODO These are strings according to the spec??
+        pub steps: u64,
+        pub memory_holes: u64,
+        pub range_check_builtin: u64,
+        pub pedersen_builtin: u64,
+        pub poseidon_builtin: u64,
+        pub ec_op_builtin: u64,
+        pub ecdsa_builtin: u64,
+        pub bitwise_builtin: u64,
+        pub keccak_builtin: u64,
```

Comment:
Unfortunately yes. We will have to format these as hex-strings for this version of the specification. The next version of the specification will have these be integers - [PR](https://github.com/starkware-libs/starknet-specs/pull/147) for this has already been merged.